packages/faceted-search/src/queryClient/tql.js

```diff
@@ -9,7 +9,7 @@ const getBadgeQueryValues = ({ properties }) => [
 
 const getBadgesQueryValues = badges => badges.map(getBadgeQueryValues);
 
-const isNotEmptyOrNull = value => value && value.length;
+const isNotEmptyOrNull = value => !!(typeof value === 'number' || (value && value.length));
```

Comment:
I'm not sure about your condition...
```javascript
const isNotEmptyOrNull = value => typeof value === 'number' || (value && value.length);
```
☝️ seems to do the job, no ? I may miss a point...

Reply:
You're right 👍  I make it too complicated, just fix that, thanks